cobra/cmd/license_gpl_3.go

```diff
@@ -18,23 +18,22 @@ package cmd
 func initGpl3() {
 	Licenses["gpl3"] = License{
 		Name:            "GNU General Public License 3.0",
-		PossibleMatches: []string{"gpl3", "gpl", "gnu gpl3", "gnu gpl"},
+		PossibleMatches: []string{"gpl3", "gplv3", "gpl", "gnu gpl3", "gnu gpl"},
```

Comment:
FWIW, a bare gpl or gnu gpl would be a "GPL 1.0 or later, but never a GPL 3.0. The GPL's are clear on the topic: no version means any version hence GPL 1.0 or later